pco_c/Cargo.toml

```diff
@@ -8,5 +8,6 @@ name = "cpcodec"
 crate-type = ["dylib"]
 
 [dependencies]
+half = { version = "2.4.1", features = ["std"] }
```

Comment:
I don't think we should need this explicitly?

Reply:
The `half` dep is needed for compilation. See other reply.